### PR TITLE
Can browse all requests

### PR DESCRIPTION
**Pull request by PaulineBA**
### User Story
```
As a visitor,
In order to find things I can help with,
I want to be able to view requests
```
## Packages/Gems added
-Active model serializers

## What we have added: 
 Add Request Index endpoint

### CODE REVIEW

**Review by ViaMarcus — APPROVED** [review]

**Review by erikbjoern — APPROVED** [review]

**Review by designerofthing — APPROVED** [review]

File: app/controllers/api/requests_controller.rb [hunk 1]
```diff
@@ -14,6 +14,11 @@ def create
     render json: { message: e.message }, status: 422
   end
 
+  def index
+    requests = Request.all
+    render json: { requests: requests.map { |req| Request::IndexSerializer.new(req) } }
```
Comment (tochman) [inline app/controllers/api/requests_controller.rb:19]: Why not just use ‘each_serializer’?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,23 @@
...
+The get request will return all the requests in the database.
+The format is:
+```
+{
+  requests: [
+    {
+      id: 1,
+      title: "Title",
+      description: "Lots of text",
+      requester: "requester@mail.com"
```
Comment (tochman) [inline README.md:18]: That is not what your serializer returns

**Review by tochman — COMMENTED** [review]
It feels incomplete. I assume that you have the associated objects created in a separate PR

File: app/controllers/api/requests_controller.rb (hunk 1 above)
Comment (ViaMarcus) [reply app/controllers/api/requests_controller.rb:19]: Each serializer did not wrap the the array in requests: for some reason.
In order to allow for easier additions of further items into the response, we did this to get the wrapping.
If you have a suggestion on a better way to do it please share.

File: README.md (hunk 2 above)
Comment (ViaMarcus) [reply README.md:18]: In the actual response the keys are wrapped as "key".
Is there a functional difference?

**Review by kermit-klein — APPROVED** [review]

**Pull request merged**